web-client/iron-remote-gui/.eslintrc.cjs

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+	root: true,
+	parser: '@typescript-eslint/parser',
+	extends: [
+		'eslint:recommended',
+		'plugin:@typescript-eslint/recommended',
+		'prettier',
+		'plugin:svelte/recommended'
+	],
+	plugins: ['@typescript-eslint'],
+	ignorePatterns: ['*.cjs'],
+	overrides: [
+		{
+			files: ['*.svelte'],
+			parser: 'svelte-eslint-parser',
+			parserOptions: {
+				parser: '@typescript-eslint/parser'
+			}
+		}
+	],
+	parserOptions: {
+		sourceType: 'module',
+		ecmaVersion: 2020,
+		project: './tsconfig.json',
+		extraFileExtensions: ['.svelte']
+	},
+	env: {
+		browser: true,
+		es2017: true,
+		node: true
+	},
+	rules: {
+		'@typescript-eslint/no-unused-vars': ['error', { argsIgnorePattern: '^_' }],
+		'svelte/valid-compile': 'off'
```

Comment:
https://github.com/sveltejs/eslint-plugin-svelte/issues/602

Raised a issue at eslint-plugin-svelte

Reply:
fixed, thanks for svelte team's fast response